components/director/pkg/graphql/schema.graphql

```diff
@@ -456,6 +456,15 @@ input LabelFilter {
     operator: FilterOperator = ALL
 }
 
+type LabelDefinition {
+    key: String!
+    schema: String
```

Comment:
Shouldn't be `CLOB` in this case?

Reply:
IMO it is better to have a string here, because it is definitely simpler, and this schema will be usually much smaller than `ApiSpec.Data`